src/components/EmailNotifications/VerifyEmailAddress.js

```diff
@@ -42,165 +42,155 @@ const VerifyEmailAddress = React.memo(function VerifyEmailAddress({
     }
   }, [])
 
+  const handleOnTermsChange = useCallback(
+    checked => {
+      setTermsAccepted(checked)
+    },
+    [setTermsAccepted]
```

Comment:
Small nitpick, we don't need to memoize this. In fact, we could pass `setTermsAccepted()` directly into the child component's handlers, but agree that it's better to have a wrapper as it's clearer.